### PR TITLE
PDF Edits

### PR DESCRIPTION
**Pull request by dciceman**
#### What's this PR do?
Updated solution to create a unique name for each generated PDF. The generated pdf document only display yes/no for any Boolean values. Fixed conditional logic error in the Employer and Wage Data sections. Added labels for each field in the Source Employer sections.
#### How should this be manually tested?
Create and submit new applications to generate application PDF document.
#### What are the relevant tickets?
PDF Edits #534

### CODE REVIEW

File: DOL.WHD.Section14c.Api/Controllers/ApplicationController.cs [hunk 1]
```diff
@@ -140,13 +140,13 @@ public ApplicationController(IIdentityService identityService, IApplicationServi
             var employerEmailTemplatePath = System.Web.Hosting.HostingEnvironment.MapPath(@"~/App_Data/EmployerEmailTemplate.txt");
             var employerEmailTemplateString = File.ReadAllText(employerEmailTemplatePath);
             var emailContents = _emailService.PrepareApplicationEmailContents(submission, certificationTeamEmailTemplateString, employerEmailTemplateString, EmailReceiver.Both);
+            var pdfName = string.Format("14c_Application_Submission_{0}_{1}_{2}.pdf", submission.Employer.LegalName, submission.Employer.PhysicalAddress.State, DateTime.Now.Year);
```
Comment (mmurthydol) [inline DOL.WHD.Section14c.Api/Controllers/ApplicationController.cs]: @dciceman David, can you provide a sample file name to see how long this is ending up to be and how it looks? Thanks!

File: DOL.WHD.Section14c.Api/Web.config [hunk 2]
```diff
@@ -22,7 +22,7 @@
     <add key="PasswordExpirationDays" value="90" /> <!-- Number of days before user password expires. A value of 0 disables password expiration -->
     <add key="AccessTokenExpireTimeSpanMinutes" value="20160" /> <!-- 14 days -->
     <add key="AttachmentRepositoryRootFolder" value="C:\temp\DOL-WHD-Section14c-Attachments\" />
-    <add key="AllowedFileNamesRegex" value="^.*\.(doc|docx|xls|pdf|jpg|jpeg|png|csv)$" />
```
Comment (mmurthydol) [inline DOL.WHD.Section14c.Api/Web.config]: Looks beautiful ;)

File: DOL.WHD.Section14c.Api/Controllers/ApplicationController.cs (hunk 1 above)
Comment (dciceman) [reply DOL.WHD.Section14c.Api/Controllers/ApplicationController.cs]: 14c_Application_Submission_My Test Employer_AL_2018.pdf

Comment (mmurthydol) [reply DOL.WHD.Section14c.Api/Controllers/ApplicationController.cs]: @david if you could please make this: 14c_Application_AL_2018-01-08_My-Test-Employer.pdf 
basically, spaces replaced with hyphen, full date in the format; order changed so when files are stored in folder CPS folks can easily organize/view them. Thanks!

Comment (dciceman) [reply DOL.WHD.Section14c.Api/Controllers/ApplicationController.cs]: @mmurthydol name format updated.

Comment (mmurthydol) [reply DOL.WHD.Section14c.Api/Controllers/ApplicationController.cs]: Thanks David! Looks great :) All good from my perspective for the PR. I haven't checked anything that Harun/Greg might check like Handlebars - those changes looked straightforward though.

**Pull request merged**